cdap-ui/app/services/constants.js

```diff
@@ -31,5 +31,32 @@ angular.module(PKG.name + '.services')
         'transform': 'transform'
       }
     },
-    addHydratorAppLabel: 'Hydrator Pipeline'
+
+    'en': {
+      hydrator: {
+        appLabel: 'Hydrator Pipeline',
+        studio: {
+          oneSinkError: 'Please add at least one sink to your pipeline.',
+          sourceRequiredFieldsError: 'Please provide required fields for Source.',
+          sinkRequiredFieldsError: 'Please provide required fields for Sink.',
+          transformRequiredFieldsError: 'Please provide required fields for Transform.',
+          oneSourceError: 'Pipelines can only have one source. Please remove any additional sources.',
```

Comment:
We should pick either "source" or "Source" (same for Transform, Sink) and use it consistently throughout these messages.